scripts/build-keystone.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Get the directory where the script is located
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Change to the root directory
+cd "$SCRIPT_DIR/.."
+
+# Load environment variables from .env file
+if [ -f veterans/.env ]; then
+    export $(cat veterans/.env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)